labels.json

```diff
@@ -2196,6 +2321,11 @@
                 "lang": "en",
                 "name": "Badger",
                 "description": "This user is a Badger! [Category: Mustelid]"
+            },
+            {
+                "lang": "pt-BR",
+                "name": "Texugo",
+                "description": "Este usuário é um Texugo!  AKA: Badger"
```

Comment:
If possible, please include  `[Category: Mustelid]` (or if there is a translation of Mustelid, then `[Category: translation_here]`